src/viam/components/movement_sensor/service.py

```diff
@@ -114,8 +114,14 @@ async def GetAccuracy(self, stream: Stream[GetAccuracyRequest, GetAccuracyRespon
         name = request.name
         sensor = self.get_resource(name)
         timeout = stream.deadline.time_remaining() if stream.deadline else None
-        accuracy = await sensor.get_accuracy(extra=struct_to_dict(request.extra), timeout=timeout, metadata=stream.metadata)
-        response = GetAccuracyResponse(accuracy=accuracy)
+        accuracy = await sensor.get_accuracy(timeout=timeout, metadata=stream.metadata)
+        response = GetAccuracyResponse(
```

Comment:
You shouldn't need to create a `GetAccuracyResponse`, you should be able to use the `accuracy` return value from `await sensor.get_accuracy(...)`